.travis.yml

```diff
@@ -1,25 +1,28 @@
 language: python
 python:
-  - "2.7"
-  - "3.6"
-  - "3.7"
-  - "3.8"
+  - 2.7
+  - 3.6
+  - 3.7
+  - 3.8
```

Comment:
Advance warning for October 2021: `3.10` will be interpreted as a float, and it'll try and (and fail) to test Python 3.1!

Demo: put this in http://www.yamllint.com/

```yaml
python:
  - 3.6
  - 3.7
  - 3.8
  - 3.9
  - 3.10
```

And watch it come out as:

```yaml
--- 
python: 
  - 3.6
  - 3.7
  - 3.8
  - 3.9
  - 3.1
```

(For fun, see also the [Norway Problem](https://hitchdev.com/strictyaml/why/implicit-typing-removed/)).

Reply:
Thanks for the warning, that made me laugh :).  I'll change this to a string instead.